features/attached_commands.feature

```diff
@@ -325,3 +325,33 @@ Feature: Command behaviour when attached to an UA subscription
            | focal   |
            | trusty  |
            | xenial  |
+
+    @series.all
+    Scenario Outline: Purge package after attaching it to a machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        And I run `touch /etc/apt/preferences.d/ubuntu-esm-infra` with sudo
+        Then I verify that running `test -f /var/log/ubuntu-advantage.log` `with sudo` exits `0`
+        And I verify that running `test -d /var/lib/ubuntu-advantage` `with sudo` exits `0`
+        And I verify that running `test -f /etc/apt/auth.conf.d/90ubuntu-advantage` `with sudo` exits `0`
+        And I verify that running `test -f /etc/apt/trusted.gpg.d/ubuntu-advantage-esm-infra-trusty.gpg` `with sudo` exits `0`
```

Comment:
Can we somehow test with globbing or ls ubuntu-advantage-*.gpg we haven't leaked any other gpg files after purge.